tools/cellprofiler/display_data_on_image.xml

```diff
@@ -0,0 +1,283 @@
+                        <option value="Object_Number">Object_Number</option>
+                    </param>
+                </when>
+                <when value="Location">
+                    <param name="obj_measurement" label="Measurement" type="select">
+                        <option value="Center_X">Center_X</option>
+                        <option value="Center_Y">Center_Y</option>
+                        <option value="Center_Z">Center_Z</option>
+                    </param>
+                </when>
+            </conditional>
+        </xml>
+
+        <xml name="display_image">
+            <conditional name="con_obj_measurement_category">
+                <param name="measurement_category" label="Measurement category" type="select">
+                    <option value="Count">Count</option>
+                    <option value="FileName">FileName</option>
+                    <option value="Frame">Frame</option>
+                    <option value="Height">Height</option>
+                    <option value="MD5Digest">MD5Digest</option>
+                    <option value="Metadata">Metadata</option>
+                    <option value="PathName">PathName</option>
+                    <option value="Scaling">Scaling</option>
+                    <option value="Series">Series</option>
+                    <option value="PathName">PathName</option>
+                    <option value="Threshold">Threshold</option>
+                    <option value="URL">URL</option>
+                    <option value="Width">Width</option>
+                </param>
+                <when value="Count">
+                    <param name="measurement" label="Measurement" type="text" />
+                </when>
+                <when value="FileName">
+                    <param name="measurement" label="Measurement" type="text" />
+                </when>
+                <when value="Frame">
+                    <param name="measurement" label="Measurement" type="text" />
+                </when>
+                <when value="Height">
+                    <param name="measurement" label="Measurement" type="text" />
+                </when>
+                <when value="MD5Digest">
+                    <param name="measurement" label="Measurement" type="text" />
+                </when>
+                <when value="Metadata">
+                    <param name="measurement" label="Measurement" type="select">
+                        <option value="FileLocation">FileLocation</option>
+                        <option value="Frame">Frame</option>
+                        <option value="ImageId">ImageId</option>
+                        <option value="Screen">Screen</option>
+                        <option value="Series">Series</option>
+                    </param>
+                </when>                                        
+                <when value="PathName">
+                    <param name="measurement" label="Measurement" type="text" />

```

Comment:
text params are in general dangerous, can we add a validator t those? For example to only allow letters and numbers?